README.md

```diff
@@ -58,9 +91,29 @@ poetry shell
 pre-commit install
 ```
 
-## Initialise shell environment
+---
+
+## Quick Run
 
-To use `nrlf` shell script commands. We must initialise the shell environment. Ensure all packages are installed, run the following commands at the root of the repository.
+For those wanting to get up and running quickly can follow this list of instructions which by the end will have given you your own workspace with the current version of NRLF running - you can find more information on steps below this section.
+
+```shell
+poetry shell
+source nrlf.sh
+nrlf make build
+nrlf aws reset-creds
+nrlf aws login mgmt <mfa>
+nrlf truststore pull-server dev
+nrlf truststore pull-client dev
+nrlf terraform plan <yourname>-test
```

Comment:
Make sure terraform is installed! Add onto the install reqs 😅

Reply:
the commends i used are in the terraform docs [here](https://developer.hashicorp.com/terraform/downloads), but the main commands are:
```
wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | sudo tee /usr/share/keyrings/hashicorp-archive-keyring.gpg
 echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
 sudo apt update && sudo apt install terraform
```